### PR TITLE
[DT-16][risk=no] Remove Feature Flag enableDrugWildcardSearch 

### PR DESCRIPTION
**Pull request by chenchals**
Description:
---
- Removed feature flag enableDrugWildcardSearch from
  - API
  - Config files for all environments

---
**PR checklist**

- [x] This PR meets the Acceptance Criteria in the JIRA story
- [x] The JIRA story has been moved to Dev Review
- [ ] This PR includes appropriate unit tests
- [ ] I have added explanatory comments where the logic is not obvious
- [ ] I have run and tested this change locally, and my testing process is described here
- [ ] If this includes a new feature flag, I have created and linked new JIRA tickets to (a) turn on the feature flag and (b) remove it later
- [ ] If this includes an API change, I have run the E2E tests on this change against my local server with [yarn test-local](https://github.com/all-of-us/workbench/blob/master/e2e/README.md#examples) because this PR won't be covered by the CircleCI tests 
- [ ] If this includes a UI change, I have taken screen recordings or screenshots of the new behavior and notified the PO and UX designer in [Slack](https://pmi-engteam.slack.com/archives/C02MWP2RN5P)
- [ ] If this change impacts deployment safety (e.g. removing/altering APIs which are in use) I have documented these in the description
- [x] If this includes an API change, I have updated the appropriate Swagger definitions and updated the appropriate API consumers
  * AoU UI
  * [Perf tests](https://github.com/broadinstitute/mcnulty/blob/develop/src/test/scala/services/AoU.scala)
  * [Researcher Directory export](https://github.com/all-of-us/workbench/wiki/Researcher-Directory-(RDR-export))
  * Cron tasks - for Offline*Controllers
  * SumoLogic - for EgressAlert

### CODE REVIEW

File: api/src/main/java/org/pmiops/workbench/config/WorkbenchConfig.java [hunk 1]
```diff
@@ -177,11 +177,9 @@ public static class AuthConfig {
 
   public static class WgsCohortExtractionConfig {
     public String serviceAccount;
-    public String serviceAccountTerraProxyGroup;
```
Comment (freemabd) [inline api/src/main/java/org/pmiops/workbench/config/WorkbenchConfig.java:180]: Please do not remove flags that belong to workbench team. Please revert this change.

File: api/src/main/java/org/pmiops/workbench/config/WorkbenchConfig.java [hunk 2]
```diff
@@ -177,11 +177,9 @@ public static class AuthConfig {
...
     public String operationalTerraWorkspaceNamespace;
     public String operationalTerraWorkspaceName;
     public String operationalTerraWorkspaceBucket;
-    public String extractionPetServiceAccount;
```
Comment (freemabd) [inline api/src/main/java/org/pmiops/workbench/config/WorkbenchConfig.java:184]: Please do not remove flags that belong to workbench team. Please revert this change.

**Review by freemabd — COMMENTED** [review]
Removing serviceAccountTerraProxyGroup and extractionPetServiceAccount has broken the dev-up command. This breaks the command line tooling. 
Screen Shot 2022-09-27 at 12 51 30 PM

File: api/src/main/java/org/pmiops/workbench/config/WorkbenchConfig.java (hunk 1 above)
Comment (chenchals) [reply api/src/main/java/org/pmiops/workbench/config/WorkbenchConfig.java:180]: oops didnt know that

Comment (chenchals) [reply api/src/main/java/org/pmiops/workbench/config/WorkbenchConfig.java]: done

File: api/src/main/java/org/pmiops/workbench/config/WorkbenchConfig.java (hunk 2 above)
Comment (chenchals) [reply api/src/main/java/org/pmiops/workbench/config/WorkbenchConfig.java]: done

**Review by freemabd — APPROVED** [review]
Thanks
